app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ]  Based on the requirements of this project, this app should ask for email confirmation please enable  `:confirmable`  in your user.rb
- Just as the previous reviewer requested, please  ensure  your  application asks for confirmation of email, currently I can signin without confirming my email